misc/vault-integration/vault.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+if [[ -z "$VAULT_ADDR" || -z "$VAULT_NAMESPACE" || -z "$ROLE" ]]; then
+    echo "MISSING REQUIRED VARIABLES: ROLE,VAULT_NAMESPACE, VAULT_ADDR, ROLE_METADATA"
+    exit 1
+fi
+# INSTALLING VAULT
+echo "1. Installing vault..."
+curl -sL https://releases.hashicorp.com/vault/1.11.1/vault_1.11.1_linux_amd64.zip -o vault1.zip
+unzip -o vault1.zip
+./vault --version
+echo "================="
+if [[ ! -z "$VAULT_TOKEN" ]]; then
+    echo '1.1 Found VAULT_TOKEN, creating JWT role... '
+    # CREATION
+    # VAULT_TOKEN IS REQUIRED FOR CREATION
+    echo "VAULT_ADDR: ${VAULT_ADDR}"
+    echo "VAULT_NAMESPACE: ${VAULT_NAMESPACE}"
+    echo "ROLE: ${ROLE}"
+
+  ./vault write auth/jwt/role/${ROLE} - <<EOF
+{
+  "user_claim": "sub",
+  "role_type": "jwt",
+  "bound_audiences": [
+    "https://dev.env0.com",
+    "https://dev-env0.auth0.com/userinfo"
```

Comment:
let's make those host names dynamic regarding stage?

Reply:
do we need it? the tests runs only on dev...

Reply:
yes, I want to merge it to master but then we could not use that in prod orgs if we want
Please fix that in your free time then merge it
@liranfarage89

Reply:
c54c223

f1f967a